ports/atmel-samd/boards/p1am_200/mpconfigboard.h

```diff
@@ -0,0 +1,32 @@
+#define MICROPY_HW_BOARD_NAME "P1AM-200"
+#define MICROPY_HW_MCU_NAME "samd51p20"
+
+#define CIRCUITPY_MCU_FAMILY samd51
+
+#define MICROPY_HW_LED_STATUS   (&pin_PB01)
+
+#define MICROPY_HW_NEOPIXEL (&pin_PC24)
+
+// These are pins not to reset.
+// QSPI Data pins
+#define MICROPY_PORT_A (PORT_PA08 | PORT_PA09 | PORT_PA10 | PORT_PA11)
+// QSPI CS, and QSPI SCK
+#define MICROPY_PORT_B (PORT_PB10 | PORT_PB11)
+#define MICROPY_PORT_C (0)
+#define MICROPY_PORT_D (0)
+
```

Comment:
This is no longer needed. See https://github.com/adafruit/circuitpython/pull/5007.
```suggestion
```